docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+db:
+  image: mdillon/postgis:9.4
```

Comment:
This may be a good excuse for us to open an Azavea account on the Docker Registry with a PostGIS image based on the official PostgreSQL image.

The definition for this specific image can be found on the Docker Registry too: https://registry.hub.docker.com/u/mdillon/postgis/dockerfile/